cpu/saml21/periph/spi.c

```diff
@@ -97,6 +97,8 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
         case SPI_SPEED_1MHZ:
             f_baud = 1000000;
             break;
+        case SPI_SPEED_4MHZ:
+            return -1;
```

Comment:
Well, IMO we don't need to list multiple unsupported modes and rather remove them in favour of a `default -1` case. One thing in addition: The SPI driver implementations will probably be adapted soon. Anyway, what is the improvement of this PR for **you** :-) ?